qiskit_ibm_runtime/api/rest/cloud_backend.py

```diff
@@ -72,6 +72,15 @@ def pulse_defaults(self) -> Dict[str, Any]:
         url = self.get_url("pulse_defaults")
         return self.session.get(url).json()
 
+    def cost_parameters(self) -> Dict[str, Any]:
+        """Return backend cost parameters.
+
+        Returns:
+            JSON response of cost parameters.
+        """
+        response = self.session.get(f"/backends/{self.backend_name}").json()
```

Comment:
`cloud_backend` uses Qiskit Runtime API. I don't think it has a `/backends/{backend name}` endpoint.